### ISSUE TITLE
Unable to use Accessibility Inspector in Developer Tools

### ISSUE THREAD

**Issue by jamesta696** [intika/Librefox#142]
Hi, 

I'm testing some developments for Web Accessibility and I'm unable to use the Accessibility Inspector. I've tried certain steps listed here but it isn't working: https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/

![image](https://user-images.githubusercontent.com/14171094/193715930-37deec36-4e62-41ca-ab8f-09cd57dbc6a9.png)

When testing this in regular Firefox it works fine. Is there a proper fix/workaround for this issue in LibreFox?

Thanks!

**Comment by jamesta696** [comment]
Wrong Repo. Sorry
